x-pack/plugins/cases/public/components/case_view/components/custom_fields.test.tsx

```diff
@@ -160,130 +162,90 @@ describe('Case View Page files tab', () => {
     userEvent.click((await screen.findAllByRole('switch'))[0]);
 
     await waitFor(() => {
-      expect(onSubmit).toBeCalledWith([
-        {
-          type: CustomFieldTypes.TEXT,
-          key: 'test_key_1',
-          value: customFieldsConfigurationMock[0].defaultValue,
-        },
-        { type: CustomFieldTypes.TOGGLE, key: 'test_key_2', value: false },
-        customFieldsMock[2],
-        customFieldsMock[3],
-      ]);
+      expect(onSubmit).toBeCalledWith({
+        type: CustomFieldTypes.TOGGLE,
+        key: 'test_key_2',
+        value: false,
+      });
     });
   });
 
-  it('adds missing defaultValues to required text custom fields without value', async () => {
+  it('replaces new toggle field correctly', async () => {
```

Comment:
Same as the test on line 152.

Maybe nitpicking but when I read "replace", I think about custom fields not in the configuration and **replacing** what should and should not be displayed.